docs/crank-to-helix-workflow.md

```diff
@@ -0,0 +1,105 @@
+
+#### Building for Windows on Windows
+Run the following in the cloned runtime repository
+```cmd
+.\build.cmd clr+libs -c Release && .\src\tests\build.cmd release generatelayoutonly
+```
+
+#### Building for Linux on Windows
+Docker can be used to build for Linux ([see documentation](https://github.com/dotnet/runtime/blob/main/docs/workflow/building/coreclr/linux-instructions.md#build-using-docker)).
+Ensure that you have Docker installed on your machine with WSL enabled. In the below script, RUNTIME_REPO_PATH should be a full path to the repo from the root.
+```cmd
+docker run --rm `
+  -v <RUNTIME_REPO_PATH>:/runtime `
+  -w /runtime `
+  mcr.microsoft.com/dotnet-buildtools/prereqs:ubuntu-22.04 `
+  ./build.sh clr+libs -c Release && ./src/tests/build.sh release generatelayoutonly
+```
+
+### Using the Crank CLI
+After installing crank as mentioned in the prerequisites, you will be able to invoke crank using `crank` in the command line. 
+
+#### Example: Run microbenchmarks on Windows x64
+Below is an example of a crank command which will run any benchmarks with Linq in the name on a Windows x64 queue. This command must be run in the performance repository, and the runtime repository must be located next to it so that you could navigate to it with `cd ../runtime`.
+
+```
+crank --config .\helix.yml --scenario micro --profile win-x64 --variable bdnArgs="--filter *Linq*" --profile msft-internal --variable buildNumber="myalias-20230811.1"
+```
+
+An explanation for each argument:
+- `--config .\helix.yml`: This tells crank what yaml file defines all the scenarios and jobs
+- `--scenario micro`: Runs the microbenchmarks scenario
+- `--profile win-x64`: Configures crank to a local Windows x64 build of the runtime, and sets the Helix Queue to a Windows x64 queue.
+- `--variable bdnArgs="--filter *Linq*"`: Sets arguments to pass to BenchmarkDotNet that will filter it to only Linq benchmarks
+- `--profile msft-internal`: Sets the crank agent endpoint to the internal hosted crank agent
+- `--variable buildNumber="myalias-20230811.1"`: Sets the build number which will be associated with the results when it gets uploaded to our storage accounts. You can use this to search for the run results in Azure Data Explorer. This build number does not have to follow any convention, the only recommendation would be to include something unique to yourself so that it doesn't conflict with other build numbers.
+
+#### Profiles
```

Comment:
Maybe it is also worth adding here that using `--profiles` multiple times is additive and does not destroy what was inserted with a previous call. That is a fairly unintuitive way for a command line option to work.